src/selectedtests/work_items/work_items_cli.py

```diff
@@ -117,10 +117,11 @@ def create_task_mapping(ctx: Context, project: str, src_regex: str, build_regex:
 @click.pass_context
 def process_task_mappings(ctx: Context, weeks_back: int) -> None:
     """Process task mapping work items that have not yet been processed."""
-    # For task mappings, after_date is compared against evergreen version create_time, which is
-    # stored in a UTC date format that is not UTC offset-aware. So after_date does not need to be
-    # offset-aware.
-    after_date = datetime.utcnow() - timedelta(weeks=weeks_back)
+    # after_date is compared against evergreen version create_time, which is stored in a UTC date
+    # format that is UTC offset-aware. So after_date needs to be offset-aware, which is why we
+    # add tzinfo below.
+    now = datetime.utcnow().replace(tzinfo=pytz.UTC)
```

Comment:
It might be worth refactoring this code into a function given that it is used in 2 places (`process_test_mappings`) and this also  makes it testable.

Reply:
good call 👍